src/components/staffing/StaffingTable.js

```diff
@@ -26,6 +29,9 @@ export default class StaffingTable extends React.Component {
         maxHeight={750}
         headerHeight={40}
       >
+        
+        {
+        (this.props.columnOrder.includes('User') || this.props.columnOrder.includes('all')) && (
```

Comment:
Is there any way to make this a bit more readable? Maybe put the || part in its own variable

Reply:
true thanks

Reply:
didnt seem to work, will make a future ticket for that